modules/administration-guide/partials/ref_grafana-dashboards-for-the-dev-workspace-engine.adoc

```diff
@@ -0,0 +1,35 @@
+[id="ref_grafana-dashboards-for-the-dev-workspace-engine_{context}"]
+= Grafana dashboards for the {devworkspace} engine
+
+This section describes the example Grafana dashboard (see link:https://github.com/devfile/devworkspace-operator/blob/main/doc/grafana/grafana-dashboard.json[grafana-dashboard.json]) which displays metrics collected from the {devworkspace} engine.
+
+.The *DevWorkspace-specific metrics* panel
+image::monitoring/monitoring-dev-workspace-engine-devworkspace-metrics-panel.png[Grafana dashboard panels containing metrics related to `DevWorkspace startup]
+The *DevWorkspace-specific metrics* panel contains information related to `DevWorkspace` startup.
+
+* *Average workspace start time* -- the average workspace start time
```

Comment:
Probably better with a definition list?

Reply:
Could you provide an example of a definition list please? Is it a kind of table? I went with this format to be consistent with https://www.eclipse.org/che/docs/che-7/administration-guide/monitoring-che/#grafana-dashboards-for-che_che

Reply:
I think this is optional, let's leave it as-is for now

Reply:
I apologize I used an obsolete name for it, they are now called description lists:  https://docs.asciidoctor.org/asciidoc/latest/lists/description/ 

This markup is semantically more correct than a list surcharged with additional formatting.

With a description list, the line would be:

```
Average workspace start time:: the average workspace start time
```